src/components/RevealComplete.tsx

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import styled from "styled-components";
+import Popup from "./Popup";
+
+type Props = {
+  show: boolean;
+  close: () => void;
+};
+
+const RevealComplete: React.FC<Props> = (props) => {
+  return (
+    <Popup
```

Comment:
We should eventually display the images of what waifus were revealed here, but fine for a v1

Reply:
Created an issue for it 😄  #138